### ISSUE TITLE
Test and fix irc.py

### ISSUE THREAD

**Issue by swalladge** [piqueserver/piqueserver#245]
It seems this has not been tested recently - currently broken on python3 since `izip` doesn't exist in itertools anymore.

**Comment by godwhoa** [comment]
Fixed py3 issues and upgraded it to new command system in [irc-fixup](https://github.com/piqueserver/piqueserver/tree/irc-fixup) branch.
But it's still broken, it fails to connect.
```
2018-01-03 07:24:55+0530 [-] Started server...
2018-01-03 07:24:55+0530 [-] Lost connection to IRC server ([Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionLost'>: Connection to the other side was lost in a non-clean fashion.
2018-01-03 07:24:55+0530 [-] ]), reconnecting in 20 seconds
2018-01-03 07:24:55+0530 [-] Stopping factory <piqueserver.irc.IRCClientFactory object at 0x7f4f2f8be5f8>
```

**Comment by godwhoa** [comment]
Closed with #247
